### ISSUE TITLE
NTR: Skin biopsy: lipid-laden infiltrates in dermis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9228]
New term request Skin biopsy: lipid-laden infiltrates in dermis (elthat_110327143627): Lipid droplets; foamy histiocytes  UMLS: 1859005

**Comment by pnrobinson** [comment]
@MickeySegal
can you proviode the disease context?

**Comment by MickeySegal** [comment]
https://ojrd.biomedcentral.com/articles/10.1186/s13023-017-0646-9
A skin biopsy performed in 6 NLSD-M and 2 NLSD-I patients, stained with O.R.O., revealed excessive lipid droplet storage in all cases.
https://www.ncbi.nlm.nih.gov/books/NBK51754/
Histologically the skin lesions exhibit hyperkeratosis, parakeratosis, and acanthosis as well as inflammatory and lipid-laden infiltrates within the dermal papillae

**Comment by pnrobinson** [comment]
label: Dermal foam cells
def: Presence of lipid-laden macrophages (foam cells) in the inner layer of the skin (dermis).
PMID: 37333211

Comment: Chronic inflammation of infectious and non-infectious origin is often associated with the presence of foam cells, lipid-laden macrophages that exhibit impaired immune function and can contribute to pathogenesis. Foam cells (or foamy macrophages) form when, due to dysregulated metabolism, lipids accumulate beyond the homeostatic capacity of macrophages. The lipids are stored as droplets that confer a foamy appearance to the macrophages

**Comment by MickeySegal** [comment]
does this get an HPO code?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000859
- label: Dermal foam cells
- parents: [Abnormal dermis morphology (HP:0034924)](https://hpo.jax.org/app/browse/term/HP:0034924)
- pmid: PMID:37333211
- def: Presence of lipid-laden macrophages (foam cells) in the inner layer of the skin (dermis).
- comment: Chronic inflammation of infectious and non-infectious origin is often associated with the presence of foam cells, lipid-laden macrophages that exhibit impaired immune function and can contribute to pathogenesis. Foam cells (or foamy macrophages) form when, due to dysregulated metabolism, lipids accumulate beyond the homeostatic capacity of macrophages. The lipids are stored as droplets that confer a foamy appearance to the macrophages.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9228
- synonyms:
